homeassistant/components/filter/sensor.py

```diff
@@ -497,7 +507,8 @@ def __init__(
         :param upper_bound: band upper bound
         :param lower_bound: band lower bound
         """
-        super().__init__(FILTER_NAME_RANGE, precision=precision, entity=entity)
+        self._default_precision = None
+        super().__init__(FILTER_NAME_RANGE, precision, entity)
```

Comment:
I think if you remove the parameter names you need to pass the window_size argument.
```suggestion
        super().__init__(FILTER_NAME_RANGE, DEFAULT_WINDOW_SIZE, precision, entity)
```